README.md

```diff
@@ -0,0 +1,10 @@
+# Kotlin Service template
+
+## What to change
+- Package name: `app.ryss.service`
+- Main Service class: `app.ryss.service.core.Service` 
+- Service Name: `app.ryss.service.core.Service#NAME`
```

Comment:
Afaik you don't actually use the `#` to reference properties in Kotlin you just use a normal `.`